### PR TITLE
Point Domain To React Build

### PR DESCRIPTION
**Pull request by btrettin**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -72,7 +75,38 @@ jobs:
           - run:
               name: Ship Build File To S3 Bucket
               command: |
-                 cd TerraWeb && aws s3 mv ./build s3://${BucketName} --recursive --acl public-read-write
+                 aws configure set aws_access_key_id $adminAccessKey
+                 aws configure set aws_secret_access_key $adminSecretAccessKey
+                 aws configure set default.region us-east-1
+                 cd LawnNannyWeb
+                 echo $FrontendDeploymentS3
+                 aws s3 mv ./build s3://${FrontendDeploymentS3} --recursive --acl public-read-write
```
Comment (jd1388) [inline .circleci/config.yml]: Is there any way these commands can be moved to a bash script or something similar?

File: .circleci/config.yml [hunk 2]
```diff
@@ -82,6 +116,10 @@ jobs:
           SBT_VERSION: 1.0.4
       steps:
           - checkout
+          - run:
+              name: Create S3 Bucket For Code
+              command: |
+                ./createLambdaBucket.sh $LambdaS3BucketName
```
Comment (jd1388) [inline .circleci/config.yml:114]: ...especially since we're already using some build scripts

**Review by jd1388 — COMMENTED** [review]
This looks pretty good. I think moving some of the `command`s to use scripts could improve these

File: .circleci/config.yml (hunk 1 above)
Comment (sdrafahl) [reply .circleci/config.yml]: I can take a look.

**Review by sdrafahl — APPROVED** [review]

**Pull request merged**